pokemongo_bot/cell_workers/spin_nearest_fort_worker.py

```diff
@@ -20,9 +20,15 @@ def work(self):
         nearest_fort = self.get_nearest_fort()
 
         if nearest_fort:
+            dist = distance(self.position[0], self.position[1], nearest_fort['latitude'], nearest_fort['longitude'])
+
             # Move to and spin the nearest stop.
-            if MoveToFortWorker(nearest_fort, self.bot).work() == WorkerResult.RUNNING:
+            if self.config.navigator != 'path' and MoveToFortWorker(nearest_fort, self.bot).work() == WorkerResult.RUNNING:
```

Comment:
@TheSavior what you think about this? His navigator is only about walking a defined route and should not walk to forts.